File: Témata/Další/R/Cvičení 02/gamrnd_Koop.R
# Vraci vektor M x 1 nezavislych vyberu
# z gama rozdeleni dle znaceni Koop(2003)
# mu ... stredni hodnota
# nu ... pocet stupnu volnosti

gamrnd_Koop <- function(mu,nu,m){
    A <-  nu/2
    B <-  2*mu/nu
    res <- B*rgamma(m, shape = A, scale = B)

    return(res)
}

File: Témata/01_NLRM_s_prirozene_konjugovanou_apriorni_hustotou/Matlab/gamrnd_Koop.m
function results = gamrnd_Koop(mu,nu,M,N)
%Vraci matici M x N nezavislych vyberu z Gamma rozdeleni dle znaceni Koop(2003)
%mu ... stredni hodnota, nu ... pocet stupnu volnosti
A = nu/2;
B = 2*mu/nu;
results = gamrnd(A,B,M,N);
end

